### DISCUSSION TITLE
E-ink displays for chartplotter

### DISCUSSION THREAD

**Discussion by mgrouch** [bareboat-necessities/lysmarine_gen#430] [Ideas]
Anyone used some marine e-ink displays?

This one looks interesting. But I’ve never seen it personally 

https://www.maptattoo.com

**Comment by andyrbarrow** [comment]
For $799 ($699 after discount) you can buy a pretty decent waterproof 1000
NIT color touch screen.
*Andy Barrow*

**Reply by mgrouch** [reply]
I’m more interested from technology point of view. It seems feasible to make a chartplotter with modern e-ink. They come with support of 7 colors already (enough for a chart) and refresh rate is getting into range suitable for a chartplotter. Benefits are better sunlight readability and much lower power consumption.

**Reply by VladimirKalachikhin** [reply]
https://gl.kobobooks.com/products/kobo-libra-colour
$220 in Amazon

But:
```
300 PPI—black-and-white content
150 PPI—colour content
```
